src/main/java/com/uid2/admin/Main.java

```diff
@@ -162,6 +160,36 @@ public void run() {
                 }
             }
 
+            CloudPath serviceMetadataPath = new CloudPath(config.getString(Const.Config.ServiceMetadataPathProp));
+            GlobalScope serviceGlobalScope= new GlobalScope(serviceMetadataPath);
+            RotatingServiceStore serviceProvider = new RotatingServiceStore(cloudStorage, serviceGlobalScope);
+            ServiceStoreWriter serviceStoreWriter = new ServiceStoreWriter(serviceProvider, fileManager, jsonWriter, versionGenerator, clock, serviceGlobalScope);
+            try {
+                serviceProvider.loadContent();
+            } catch (CloudStorageException e) {
+                if(e.getMessage().contains("The specified key does not exist")) {
```

Comment:
This exception message looks like it is related to key sets, not services. Is this correct?

Reply:
key refers to json key in the metadata. in this case the key would be either location, services, or service_links